ccd-definition/Jurisdiction.json

```diff
@@ -3,6 +3,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "PUBLICLAW",
     "Name": "Public Law",
-    "Description": "Public Law"
+    "Description": "Public Law",
+    "Shuttered": "Yes"
```

Comment:
Just FMI, Does it shutter service when we release this?

Reply:
yeah I think so

Reply:
Should we put this nonprod for the time being?

Reply:
Can do

Reply:
Will just need a small ticket to toggle on friday

Reply:
🤔 seems that it isn't as simple as just duplicating and adding `-nonprod` to the file name as both are still read locally and there can only be one jurisdiction